components/image-builder-bob/pkg/proxy/proxy.go

```diff
@@ -185,6 +185,10 @@ func (proxy *Proxy) reverse(alias string) *httputil.ReverseProxy {
 			}
 			return true, nil
 		}
+		if resp.StatusCode == http.StatusBadRequest {
+			log.WithField("URL", resp.Request.URL.String()).Warn("bad request")
+			return true, nil
```

Comment:
The client is setup to retry up to 3 times. The default wait time in-between retries is 1s. This should help, if we're hitting transient issues that are resulting in bad requests.

Without this change, when we encounter a 400 error, we do not retry, and the image build fails.